x-pack/plugins/spaces/public/share_saved_objects_to_space/components/share_to_space_form.tsx

```diff
@@ -67,28 +78,13 @@ export const ShareToSpaceForm = (props: Props) => {
     <div data-test-subj="share-to-space-form">
       {getShareWarning()}
 
-      <EuiFormRow
-        label={
-          <FormattedMessage
-            id="xpack.spaces.management.shareToSpace.selectSpacesLabel"
-            defaultMessage="Select spaces"
-          />
-        }
-        labelAppend={
-          <FormattedMessage
-            id="xpack.spaces.management.shareToSpace.selectSpacesLabelAppend"
-            defaultMessage="{selectedCount} selected"
-            values={{ selectedCount: props.shareOptions.selectedSpaceIds.length }}
-          />
-        }
-        fullWidth
-      >
-        <SelectableSpacesControl
-          spaces={props.spaces}
-          selectedSpaceIds={props.shareOptions.selectedSpaceIds}
-          onChange={(selection) => setSelectedSpaceIds(selection)}
-        />
-      </EuiFormRow>
+      <ShareModeControl
+        coreStart={coreStart}
```

Comment:
Can be removed since it's unused by child

Reply:
[It's actually used](https://github.com/elastic/kibana/pull/76132#discussion_r498483259)